### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by tdjastrzebski** [N0ury/dmm_util#16]
```
C:\Temp>python -m fluke_28x_dmm_util -p COM6
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\__main__.py", line 16, in <module>
    cli()
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\__main__.py", line 13, in cli
    dmm_util.main()
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\dmm_util.py", line 627, in main
    match args.command[0]:
IndexError: list index out of range
```

**Comment by N0ury** [comment]
This has already been solved.
You'll find it in next commit.
I hope this week. I'm currently working on the timeout issue which in my mind is the most important issue.

**Comment by N0ury** [comment]
Should be solved in commit [c92b1b1](https://github.com/N0ury/dmm_util/commit/c92b1b16a1d5e2593059ce53a13940256d3018d4)
